### PR TITLE
*: cut v2.6.0-rc.0

### PR DESCRIPTION
**Pull request by simonpasquier**
This is a first draft for RC.0.

cc @brian-brazil @brancz @SuperQ @krasi-georgiev @fabxc @beorn7 @gouthamve

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+* [ENHANCEMENT] Kubernetes SD: Log Kubernetes messages. #4931
+* [ENHANCEMENT] Promtool: Collect CPU and trace profiles. #4897
+* [ENHANCEMENT] Promtool: Support writing output as JSON. #4848
+* [ENHANCEMENT] Remote Read: Return available data if remote read fails partially. #4832
+* [ENHANCEMENT] Remote Write: Improve queue performance. #4772
+* [ENHANCEMENT] Remote Write: Add min_shards parameter to set the minimum number of shards. #4924
+* [ENHANCEMENT] TSDB: memory improvements. #4953
+* [ENHANCEMENT] Web: Log stack traces on panic. #4221
+* [ENHANCEMENT] Web UI: Add copy to clipboard button for configuration. #4410
+* [ENHANCEMENT] Web UI: Display job label in targets page. #4806
```
Comment (brian-brazil) [inline CHANGELOG.md]: This can be merged with the above one

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+* [ENHANCEMENT] Web UI: Support console queries at specific times. #4764
+* [BUGFIX] Deduplicate handler labels for HTTP metrics. #4732
+* [BUGFIX] Fix leaked queriers causing shutdowns to hang. #4922
+* [BUGFIX] Fix configuration loading panics on nil pointer slice elements. #4942
+* [BUGFIX] API: Correctly skip mismatching targets on /api/v1/targets/metadata. #4905
+* [BUGFIX] API: Better rounding for incoming query timestamps. #4941
+* [BUGFIX] Azure SD: Fix panic. #4867
+* [BUGFIX] Console templates: Fix hover when the metric has a null value. #4906
+* [BUGFIX] Custom SD adapter: Only update the JSON file when the targets change. #4567
```
Comment (brian-brazil) [inline CHANGELOG.md]: I don't think we list these

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,46 @@
...
+* [BUGFIX] Azure SD: Fix panic. #4867
+* [BUGFIX] Console templates: Fix hover when the metric has a null value. #4906
+* [BUGFIX] Discovery: Remove all targets when the scrape configuration gets empty. #4819
+* [BUGFIX] Marathon SD: Fix leaked connections. #4915
+* [BUGFIX] Marathon SD: Use 'hostPort' member of portMapping to construct target endpoints. #4887
+* [BUGFIX] PromQL: Fix a goroutine leak in the lexer/parser. #4858
+* [BUGFIX] Scrape: Pass through content-type for non-compressed output. #4912
+* [BUGFIX] Scrape: Fix deadlock in the scrape's manager. #4894
+* [BUGFIX] Scrape: Scrape targets at fixed intervals even after Prometheus restarts. #4926
+* [BUGFIX] Web: Avoid proxy to connect to the local gRPC server. #4572
```
Comment (gouthamve) [inline CHANGELOG.md]: Missing the following:
https://github.com/prometheus/tsdb/pull/461 (Make sure backup restore works)
https://github.com/prometheus/tsdb/pull/451 (One of the WAL error handling PRs)
https://github.com/prometheus/tsdb/pull/440 (WAL reading speed up in enhancements)

Comment (simonpasquier) [reply CHANGELOG.md]: Thanks I've added them with a link to #4953

**Comment by brian-brazil** [comment]
:+1:

**Comment by simonpasquier** [comment]
Meh, didn't pick the last release to compare with...

/benchmark cancel

**Comment by simonpasquier** [comment]
No big differences between the 2 versions although the RC is a bit less demanding on memory allocations...

/benchmark cancel

**Pull request merged**